### ISSUE TITLE
popup resize

### ISSUE THREAD

**Issue by cop98** [traccar/traccar-web#1176]
Thanks to PR https://github.com/traccar/traccar-web/pull/1155 the popup has been fixed (it is a great thing compared to before!) It is a bit too small in some situations, though.
Can it be made to resize it as desired?

**Comment by cop98** [comment]
Yes I confirm that it is scrollable. I meant whether you could make it so that it could be enlarged or shrunk at will.

**Comment by tananaev** [comment]
Oh you mean manually resized?

**Comment by cop98** [comment]
Sie satto! I would leave it default as it is now but make it so that you can manually change the size
